packages/client/src/pages/game/game.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC, useRef, useEffect } from 'react';
+import style from './canvas.module.scss';
+
+const Game: FC = () => {
+    const ref = useRef<HTMLCanvasElement | null>(null);
+
+    useEffect(() => {
+        if (ref.current) {
+            const ctx = ref.current.getContext('2d');
+
+            // render main field
+            // eslint-disable-next-line
+            const roundRect = function <T extends number>(x: T, y: T, width: T, height: T, radius: T): void {
+                ctx!.beginPath();
+                ctx!.moveTo(x + radius, y);
+                ctx!.arcTo(x + width, y, x + width, y + height, radius);
+                ctx!.arcTo(x + width, y + height, x, y + height, radius);
+                ctx!.arcTo(x, y + height, x, y, radius);
+                ctx!.arcTo(x, y, x + width, y, radius);
+                ctx!.closePath();
+                ctx!.fillStyle = '#265B8F';
```

Comment:
Предлагаю не указывать цвета напрямую в коде, а лучше из variables.scss сделать экспорт переменных с цветами
`:export{
  colorMain: $colorMain;
  colorYellow: $colorYellow;
}` и использовать их в коде.
```suggestion
                ctx!.fillStyle = style.colorBlue;
```

Reply:
идея прикольная, но что-то у меня не получатся экспортировать их
Ты в каком-то блоке уже применял такую задумку?